auditlog/admin-service/src/main/java/io/debezium/demos/auditing/admin/TransactionEvent.java

```diff
@@ -0,0 +1,68 @@
+package io.debezium.demos.auditing.admin;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TransactionEvent {
+    
+    @JsonProperty("before")
```

Comment:
Tiny nit: I'd suggest to omit the annotation where the field name itself is fine.

Reply:
refactored as suggested